<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>rncbc aka Rui Nuno Capela</author>
 <comment>qtractor - An Audio/MIDI multi-track sequencer.

   Copyright (C) 2005-2017, rncbc aka Rui Nuno Capela. All rights reserved.

   This program is free software; you can redistribute it and/or
   modify it under the terms of the GNU General Public License
   as published by the Free Software Foundation; either version 2
   of the License, or (at your option) any later version.

   This program is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU General Public License for more details.

   You should have received a copy of the GNU General Public License along
   with this program; if not, write to the Free Software Foundation, Inc.,
   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.

 </comment>
 <class>qtractorMidiImportForm</class>
 <widget class="QDialog" name="qtractorMidiImportForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>0</width>
    <height>0</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MIDI import</string>
  </property>
  <property name="focusPolicy" >
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowIcon" >
   <iconset resource="qtractor.qrc" >:/images/itemMidiFile.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="Plugins">
     <property name="title">
      <string>Plugins</string>
     </property>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="qtractorPluginListView" name="PluginListView" >
         <property name="toolTip" >
          <string>Track plugins</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" >
         <item>
          <widget class="QToolButton" name="AddPluginToolButton" >
           <property name="minimumSize" >
            <size>
             <width>90</width>
             <height>28</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>Add plugin</string>
           </property>
           <property name="text" >
            <string>&amp;Add...</string>
           </property>
           <property name="icon" >
            <iconset resource="qtractor.qrc" >:/images/formCreate.png</iconset>
           </property>
           <property name="toolButtonStyle" >
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="RemovePluginToolButton" >
           <property name="minimumSize" >
            <size>
             <width>90</width>
             <height>28</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>Remove plugin</string>
           </property>
           <property name="text" >
            <string>&amp;Remove</string>
           </property>
           <property name="icon" >
            <iconset resource="qtractor.qrc" >:/images/formRemove.png</iconset>
           </property>
           <property name="toolButtonStyle" >
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>8</width>
             <height>8</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="MoveUpPluginToolButton" >
           <property name="minimumSize" >
            <size>
             <width>90</width>
             <height>28</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>Move plugin up</string>
           </property>
           <property name="text" >
            <string>&amp;Up</string>
           </property>
           <property name="icon" >
            <iconset resource="qtractor.qrc" >:/images/formMoveUp.png</iconset>
           </property>
           <property name="toolButtonStyle" >
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="MoveDownPluginToolButton" >
           <property name="minimumSize" >
            <size>
             <width>90</width>
             <height>28</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>Move plugin down</string>
           </property>
           <property name="text" >
            <string>&amp;Down</string>
           </property>
           <property name="icon" >
            <iconset resource="qtractor.qrc" >:/images/formMoveDown.png</iconset>
           </property>
           <property name="toolButtonStyle" >
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>4</number>
     </property>
     <property name="margin" >
      <number>8</number>
     </property>
     <item>
      <widget class="QGroupBox" name="InstrumentBankGroup">
       <property name="title">
        <string>Instrument and bank selection</string>
       </property>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QGroupBox" name="InstrumentGroup">
          <property name="title">
           <string>Instruments</string>
          </property>
          <layout class="QGridLayout" >
           <property name="margin" >
            <number>8</number>
           </property>
           <property name="spacing" >
            <number>4</number>
           </property>
           <item row="0" column="0" >
            <widget class="QLabel" name="InstInstTextLabel" >
             <property name="text" >
              <string>&amp;Instrument:</string>
             </property>
             <property name="buddy" >
              <cstring>InstInstComboBox</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="5" >
            <widget class="QComboBox" name="InstInstComboBox" >
             <property name="minimumSize" >
              <size>
               <width>250</width>
              </size>
             </property>            
             <property name="toolTip" >
              <string>Select instrument for all instruments</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QLabel" name="InstBankTextLabel" >
             <property name="text" >
              <string>&amp;Bank:</string>
             </property>
             <property name="buddy" >
              <cstring>InstBankComboBox</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="5" >
            <widget class="QComboBox" name="InstBankComboBox" >
             <property name="minimumSize" >
              <size>
               <width>250</width>
              </size>
             </property>            
             <property name="toolTip" >
              <string>Select bank for all instruments</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="DrumGroup">
          <property name="title">
           <string>Drums</string>
          </property>
          <layout class="QGridLayout" >
           <property name="margin" >
            <number>8</number>
           </property>
           <property name="spacing" >
            <number>4</number>
           </property>
           <item row="0" column="0" >
            <widget class="QLabel" name="DrumInstTextLabel" >
             <property name="text" >
              <string>&amp;Instrument:</string>
             </property>
             <property name="buddy" >
              <cstring>DrumInstComboBox</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="5" >
            <widget class="QComboBox" name="DrumInstComboBox" >
             <property name="minimumSize" >
              <size>
               <width>250</width>
              </size>
             </property>            
             <property name="toolTip" >
              <string>Select instrument for all drums</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QLabel" name="DrumBankTextLabel" >
             <property name="text" >
              <string>&amp;Bank:</string>
             </property>
             <property name="buddy" >
              <cstring>DrumBankComboBox</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="5" >
            <widget class="QComboBox" name="DrumBankComboBox" >
             <property name="minimumSize" >
              <size>
               <width>250</width>
              </size>
             </property>            
             <property name="toolTip" >
              <string>Select bank for all drums</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="TrackNameGroup">
     <property name="title">
      <string>Track names</string>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QRadioButton" name="NameMidiRadioButton">
        <property name="toolTip">
         <string>Set MIDI filename (without extension) for track name</string>
        </property>
        <property name="text">
         <string>&amp;MIDI filename</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="NameTrackRadioButton">
        <property name="toolTip">
         <string>Set 'Track n' (n = number) for track name</string>
        </property>
        <property name="text">
         <string>&amp;Track [n]</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="NamePatchRadioButton">
        <property name="toolTip">
         <string>Instrument name</string>
        </property>
        <property name="text">
         <string>&amp;Patch name</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>      
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="margin">
      <number>8</number>
     </property>
     <item>
      <widget class="QDialogButtonBox" name="DialogButtonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="8" />
 <customwidgets>
  <customwidget>
   <class>qtractorPluginListView</class>
   <extends>QListWidget</extends>
   <header>qtractorPluginListView.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>PluginListView</tabstop>
  <tabstop>AddPluginToolButton</tabstop>
  <tabstop>RemovePluginToolButton</tabstop>
  <tabstop>MoveUpPluginToolButton</tabstop>
  <tabstop>MoveDownPluginToolButton</tabstop>
  <tabstop>InstInstComboBox</tabstop>
  <tabstop>InstBankComboBox</tabstop>
  <tabstop>DrumInstComboBox</tabstop>
  <tabstop>DrumBankComboBox</tabstop>
  <tabstop>NameMidiRadioButton</tabstop>
  <tabstop>DialogButtonBox</tabstop>
 </tabstops>
 <resources>
  <include location="qtractor.qrc" />
 </resources>
 <connections/>
</ui>
